src/app/shared/learning-object/learning-object.component.ts

```diff
@@ -114,14 +114,14 @@ export class LearningObjectListingComponent implements OnInit, OnChanges {
   //Method for title case inside tool tip
   toTitleCase = function (str) {
     str = str.toLowerCase().split(' ');
-    for (var i = 0; i < str.length; i++) {
+    for (let i = 0; i < str.length; i++) {
       str[i] = str[i].charAt(0).toUpperCase() + str[i].slice(1);
     }
     return str.join(' ');
   };
 
   getContributors() { 
-    return this.learningObject.contributors.map(x => this.toTitleCase(x.name + ' '));
+    return this.learningObject.contributors.map(x => this.toTitleCase(' ' + x.name));
```

Comment:
Won't this add extra spaces between names? Additionally, adding the space at index 0 of the string, means the logic in the above function (`str[I].charAt(0)`) will attempt to capitalize the space instead of capitalizing the first letter of the word.

Reply:
Not necessarily, because the extra space I concatenated with the name is to change the position of the comm

a. Without the space added the names of the contributors will be squished together like so and you can see that the title case method still works the same way ![withoutspace](https://user-images.githubusercontent.com/37280445/53030195-5deae980-3438-11e9-915a-2b3f41b244a6.PNG)